lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -157,7 +209,8 @@ shutdown_unknown:
   default_value: true
 
 shutdown_vlan:
-  default_value: true
+  nxapi:
+    default_value: true
```

Comment:
Currently it will create an 'empty' CmdRef object - so `CommandReference.lookup('interface', 'shutdown_vlan')` will not raise an error, but the resulting CmdRef object won't have any attributes that can be referenced.

This is something that will need to be improved as part of the broader changes we discussed with Deepak on Friday.